optimize-onnx/src/main.cpp

```diff
@@ -0,0 +1,58 @@
+#include <fstream>
+
+#include "export.h"
+#include "import.h"
+
+
+// reads a protobuf file on stdin and writes it back to stdout.
+// doc_strings are lost, but semantics should be unchanged.
+int main(int argc, char** argv) {
+  std::stringstream buffer;
+  buffer << std::cin.rdbuf();
```

Comment:
this might a problem with reading binary data: https://stackoverflow.com/questions/7587595/read-binary-data-from-stdcin or does rdbuf() solve all of that?